src/fluree/db/flake/flake_db.cljc

```diff
@@ -283,6 +284,8 @@
       (when (empty? all-flakes)
         (commit-error "Commit has neither assertions or retractions!"
                       commit-metadata))
+      (log/debug "Updating db" (str/join "@" [(:alias db) (:t db)])
```

Comment:
I'm not sure `str/join` really brings any benefit over `str` when we're always joining two strings.

Reply:
They're about the same. `str/join` is just the function I prefer to use when joining any number of strings with the same delimiter.